tests/README.md

```diff
@@ -50,6 +50,10 @@ To run the same tests on a release build, replace `debug` with `release`.
 # Functional tests
 
 [TODO]
+Functional tests are located under the `tests/functional` directory. 
+Requires installing: https://github.com/emesik/monero-python
```

Comment:
How needed is this ? Needing some random external code to run tests seems dangerous.

Reply:
It is just a small wrapper for rpc calls. I will message the current maintainer to ask whether any written consent is needed to copy it, and otherwise rewrite from scratch.